includes/core-changes/corefx/2.1/process-start-changes.md

```diff
@@ -6,6 +6,8 @@
 
 <xref:System.Diagnostics.Process.Start%2A?displayProperty=nameWithType> lets you launch an application directly, for example, with code such as `Process.Start("mspaint.exe")` that launches Paint. It also lets you indirectly launch an associated application if <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> is set to `true`. On .NET Framework, the default value for <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> is `true`, meaning that code such as `Process.Start("mytextfile.txt")` would launch Notepad, if you've associated *.txt* files with that editor. To prevent indirectly launching an app on .NET Framework, you must explicitly set <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> to `false`. On .NET Core, the default value for <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> is `false`. This means that, by default, associated applications are not launched when you call `Process.Start`.
 
+The following properties on <xref:System.Diagnostics.ProcessStartInfo?displayProperty=nameWithType> are only functional when <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> is `true`: <xref:System.Diagnostics.ProcessStartInfo.CreateNewWindow?displayProperty=nameWithType>, <xref:System.Diagnostics.ProcessStartInfo.ErrorDialog?displayProperty=nameWithType>, <xref:System.Diagnostics.ProcessStartInfo.Verb?displayProperty=nameWithType> and <xref:System.Diagnostics.ProcessStartInfo.WindowsStyle?displayProperty=nameWithType>. To use these properties on .NET Core, you must explicitly set <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> to `true`.
```

Comment:
This should be a list. I also removed `To use these properties on .NET Core, you must explicitly set <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> to true.`  as I'm unsure why this clause is there since the first paragraph mentions that these properties only work when it's set to true. Does the .NET Core part matter?

```suggestion
The following properties on <xref:System.Diagnostics.ProcessStartInfo?displayProperty=nameWithType> are only functional when <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute?displayProperty=nameWithType> is `true`:

- <xref:System.Diagnostics.ProcessStartInfo.CreateNewWindow?displayProperty=nameWithType>
- <xref:System.Diagnostics.ProcessStartInfo.ErrorDialog?displayProperty=nameWithType>
- <xref:System.Diagnostics.ProcessStartInfo.Verb?displayProperty=nameWithType>
- <xref:System.Diagnostics.ProcessStartInfo.WindowsStyle?displayProperty=nameWithType>.
```

Reply:
> Does the .NET Core part matter?

In the linked issue, the user was doing this:
```
            emulatorProcess = new Process();
            emulatorProcess.StartInfo.FileName = @"c:\windows\System32\notepad.exe";
            emulatorProcess.StartInfo.WindowStyle = ProcessWindowStyle.Minimized;
            emulatorProcess.Start();
```
This works on .NET Framework but not on .NET Core. This is because WindowStyle (and the other three properties listed above) only work when UseShellExecute is true. On .NET Framework, one would not know this since it defaults to true. This means that code like the above fails on .NET Core but it won't be obvious that the fix is to set UseShellExecute to true. This is what I was trying to describe.

It's unfortunate BTW that they don't throw an exception when you attempt to use them without UseShellExecute, but they don't.